topology/University of Nebraska/Nebraska-Lincoln/UNLCachingInfrastructure.yaml

```
@@ -169,3 +169,32 @@ Resources:
         Description: HCC StashCache Origin Server
     AllowedVOs:
       - HCC
+      
+    LIGO-StashCache-Origin:
```

@hzhu16 it looks like the indentation is off on this line:

```suggestion
  LIGO-StashCache-Origin:
```